xwiki-rendering-syntaxes/xwiki-rendering-syntax-xwiki20/src/main/java/org/xwiki/rendering/internal/renderer/xwiki20/XWikiSyntaxChainingRenderer.java

```diff
@@ -307,6 +307,8 @@ public void endFormat(Format format, Map<String, String> parameters)
                 break;
             case NONE:
                 break;
+            default : //none of the case matched
```

Comment:
A more specific comment would be nicer :)

Like "Unsupported format" for example.

Reply:
ok